MapboxNavigation/NavigationViewController.swift

```diff
@@ -276,8 +276,8 @@ public class NavigationViewController: UIViewController, RouteMapViewControllerD
     var styleTypeForTimeOfDay: StyleType {
         guard automaticallyAdjustsStyleForTimeOfDay else { return .dayStyle }
 
-        guard let location = routeController.location,
-            let solar = Solar(coordinate: location.coordinate),
+        guard let coordinate = routeController.location?.coordinate,
+            let solar = Solar(latitude: coordinate.latitude, longitude: coordinate.longitude),
```

Comment:
This change should not happen, the api changed when we updated to swift 4/solar 2.1.

Reply:
-  [ ] Revert Change